.github/workflows/ci_workflows.yml

```diff
@@ -4,9 +4,9 @@ on: [push, pull_request]
 
 jobs:
   retrieve_cache:
-    uses: spacetelescope/stpsf/.github/workflows/retrieve_cache.yml@098e3d3515d906bbfa12680d2282ec8d6726527b  # v1.5.2
+    uses: spacetelescope/stpsf/.github/workflows/retrieve_cache.yml@1b4adc279d0e3c76e01d975fb2a46a10258b5d53 # setup retrieve_cache with development or latest_release
```

Comment:
make sure this commit hash is available on `develop` after this branch is merged!

Reply:
Hash's should be retained through commit history so should be good to go